test/Main.java

```diff
@@ -97,6 +107,7 @@ private static boolean isValidChoice(int choice) {
                 return true;
             }
         }
+        System.out.println("Invalid Choice! Please choose again!");

```

Comment:
không để in ra màn hình ở đây đưa vào trong main

Reply:
là sao nhỉ